integration-test/support/steps_file.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+// in this file you can append custom step methods to 'I' object
+
+module.exports = function() {
+  return actor({
+    // Define custom steps here, use 'this' to access default methods of I.
+    // It is recommended to place a general 'login' function here.
+    loginAs(username, password) {
+      this.fillField("Username", username);
+      this.fillField("Password", password);
+      this.click("Login");
+    },
+
+    loginForTest() {
+      this.amOnPage("http://localhost:3000");
+      this.loginAs("John", "Snow");
```

Comment:
Do we need to setup our MongoDBs to be able to run the integration tests?

Reply:
Yes, the user is being added via `npm user-add` before the tests run (is done in `travis-integration-test.sh`)